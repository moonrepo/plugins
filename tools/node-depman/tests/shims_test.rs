#[cfg(not(windows))]
mod npm {
    use proto_pdk_test_utils::*;

    generate_shims_test!("npm-test", ["npm", "npx", "node-gyp"]);
}

#[cfg(not(windows))]
mod pnpm {
    use proto_pdk_test_utils::*;

    generate_shims_test!("pnpm-test", ["pnpm", "pnpx"]);
}

#[cfg(not(windows))]
mod yarn {
    use proto_pdk_test_utils::*;

    generate_shims_test!("yarn-test", ["yarn", "yarnpkg"]);
}
