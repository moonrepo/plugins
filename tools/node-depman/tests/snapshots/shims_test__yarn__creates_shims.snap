---
source: tools/node-depman/tests/shims_test.rs
expression: "std :: fs ::\nread_to_string(sandbox.path().join(\".proto/shims/registry.json\")).unwrap()"
---
{
  "yarn": {},
  "yarnpkg": {
    "alt_bin": true,
    "parent": "yarn-test"
  }
}
